docs/guides/vtex-io/Storefront-Guides/storefront-best-practices/storefront-analytics.md

```diff
@@ -0,0 +1,46 @@
+
+The integration with Google Analytics is available for all VTEX stores. Learn more in [Configuring Google Analytics 4 in VTEX stores](https://help.vtex.com/en/tutorial/how-to-setup-google-analytics-in-vtex-store--G2P0rmSrEiqCcmUMyUUwG).
+
+## Implementing Google Analytics
+
+Integrating Google Analytics with your VTEX storefront allows you to track key ecommerce metrics and customer behavior, such as product views, cart activity, and checkout behavior, directly within the VTEX platform.
+
+To learn how to implement Google Analytics, see the section for your store's storefront solution.
+
+### Store Framework
+
+In [Store Framework](https://developers.vtex.com/docs/guides/store-framework), a VTEX IO app manages Google Analytics by implementing the [Google Tag Manager (GTM)](https://tagmanager.google.com/) script, setting event properties, and adding events to the `dataLayer`.
+
+> ℹ️ Learn more in the guide [Analytics on Store Framework](https://developers.vtex.com/docs/guides/storefront-analytics-on-store-framework).
+
+Below are the main characteristics of Google Analytics in Store Framework:
+
+- It's easy to handle and maintain, as everything is configured to automatically track and send data to GTM by using the native Google Tag Manager app. However, deviating from the predefined settings requires significant effort and customization, often defeating the framework's purpose.
+- Provides analytics events for free when using the native Google Tag Manager app and native blocks to compose your pages. In this case, you do not need to actively manage analytics, but this convenience comes with limitations and less control over customization.
+- Identifying where or when an event is triggered can be challenging, making debugging and customization more challenging. You rely on the app's predefined events and DOM interactions, which may limit visibility and control.
+
+### FastStore
+
+In [FastStore](https://developers.vtex.com/docs/guides/faststore), there is no equivalent app; instead, all management is done directly within the storefront source code.
+
+> ℹ️ Learn more in the guide [Analytics on FastStore](https://developers.vtex.com/docs/guides/storefront-analytics-on-faststore).
+
+Below are the main characteristics of Google Analytics in FastStore:
+
+- You can trigger your own events, customize them with your own types, and add any properties of your choice. This flexibility empowers developers and store managers to collect data tailored to their business needs. Learn how to extend and customize types in the guide [Extending and customizing native types](https://v1.faststore.dev/reference/sdk/analytics/how-to-extend-types). For more information about sending custom events, see [Sending custom events](https://v1.faststore.dev/reference/sdk/analytics/how-to-send-custom-events).
```

Comment:
Essas documentações de _Extending and customizing native types_ e _Sending custom events_ ainda estão em revisão com o time. Elas existem no portal da FastStore v1, mas ele está fora do ar desde ontem. O time já reportou para traffic, mas ainda não voltou. Uma ideia é retirar (por enquanto) essas duas partes que mencionam essas docs ("Learn how to extend and customize types in...") ou manter o link da v1, mas não tenho certeza quando o portal estará novamente no ar. O que acha @carolinamenezes ?